src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/client/payhub/request/CreatePaymentRequest.java

```diff
@@ -18,8 +18,8 @@
     @JsonProperty("site_id")
     public final String siteId;
 
-    public PaymentRequest(String ccdCaseNumber, List<String> documentControlNumbers,
-                          boolean isExceptionRecord, String siteId) {
+    public CreatePaymentRequest(String ccdCaseNumber, List<String> documentControlNumbers,
```

Comment:
we can format this - each argument on new line ?

Reply:
formatted